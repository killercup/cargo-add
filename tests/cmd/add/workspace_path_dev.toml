bin.name = "cargo-add"
args = ["add", "cargo-list-test-fixture-dependency", "../dependency", "--dev"]
status = "success"
stdout = ""
stderr = """
      Adding cargo-list-test-fixture-dependency (unknown version) to dev-dependencies.
      Adding cargo-list-test-fixture-dependency (unknown version) to dev-dependencies.
"""
fs.sandbox = true
fs.cwd = "workspace_path_dev.in/primary"

[env.add]
CARGO_IS_TEST="1"
