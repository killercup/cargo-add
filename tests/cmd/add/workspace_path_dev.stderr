      Adding cargo-list-test-fixture-dependency (unknown version) to dev-dependencies. 
      Adding cargo-list-test-fixture-dependency (unknown version) to dev-dependencies. 
